src/components/Card/CardTitle.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component, PropTypes } from 'react'
+import { View, Text } from 'react-native'
+import Uranium from 'uranium'
+import { Breakpoints, connectTheme } from '../../index'
+
+class CardTitle extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    subtitle: PropTypes.node,
+    subtitleStyle: PropTypes.object,
+    title: PropTypes.node,
+    titleStyle: PropTypes.object,
+    theme: PropTypes.object.isRequired,
+    style: PropTypes.oneOfType([
```

Comment:
Man, I gotta write contributing guidelines, realizing how many OCD rules I have haha.

children/style at the bottom 'cause they're not as important. Props that are unique to this component go at the top 'cause they're the most important. Also, add comment if prop comes from a higher-order function. See other components